sql/src/test/java/io/crate/operation/merge/AbstractBucketMergerTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to CRATE Technology GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.operation.merge;
+
+import com.google.common.util.concurrent.Futures;
+import com.google.common.util.concurrent.ListenableFuture;
+import io.crate.core.collections.Bucket;
+import io.crate.core.collections.CollectionBucket;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class AbstractBucketMergerTest {
+    protected List<ListenableFuture<Bucket>> createBucketFutures(List<Object[]> ... buckets) {
```

Comment:
can you make this static and move into a BucketTestingHelpers class instead of subclassing?